packages/api/src/external/fhir/document/index.ts

```diff
@@ -62,7 +62,17 @@ export function createDocReferenceContent({
   return content;
 }
 
-function getFormat(format: string | string[] | undefined): Coding | undefined {
+function isCoding(value: unknown): value is Coding {
+  return (
+    typeof value === "object" &&
+    value !== null &&
+    !Array.isArray(value) &&
+    ("system" in value || "display" in value || "code" in value)
+  );
+}
+
+function getFormat(format: string | string[] | Coding | undefined): Coding | undefined {
+  if (isCoding(format)) return format;
```

Comment:
fairly defensive type guard that will skip if values of the old types are passed in